.github/workflows/tag-and-release.yml

```diff
@@ -95,3 +95,27 @@ jobs:
             minikube-gui-macos.dmg
             minikube-gui-macos.tar.gz
             minikube-gui-windows.zip
+  bump-releases:
```

Comment:
how about a more clear name for this ? so we dont think it bumps the actual release,

how about update release josn ?

Reply:
Good point, updated the name